### ISSUE TITLE
Missing React.unmountComponentAtNode()

### ISSUE THREAD

**Issue by maxwellpeterson-wf** [cleandart/react-dart#29]
This dart wrapper seems to be missing [`React.unmountComponentAtNode(...)`](http://facebook.github.io/react/docs/top-level-api.html#react.unmountcomponentatnode)

Error:
```
No top-level method 'react.unmountComponentAtNode' declared.
NoSuchMethodError: method not found: 'react.unmountComponentAtNode'
```

**Comment by hleumas** [comment]
Thank you for reporting! We will take a look at this issue soon.

**Comment by matystl** [comment]
implemented in 0.5.2 and published on pub
